libraries/vault_config.rb

```diff
@@ -37,6 +37,7 @@ class VaultConfig < Chef::Resource
       attribute(:tls_cert_file, kind_of: String)
       attribute(:tls_key_file, kind_of: String)
       attribute(:cache_size, kind_of: Fixnum)
+      attribute(:disable_cache, equal_to: [true, false], default: false)
```

Comment:
I'd suggest removing `default: false` here, then this option will appear in the config file only if we've set it explicitly. The default behavior won't be changed, but it will help to avoid redundant options to be rendered in the config file.

It especially makes sense for older Vault versions (< 0.4.0), where `disable_cache` option is not available.